examples/examples_fast/Modules/main.py

```diff
@@ -8,10 +8,10 @@
 
 import time
 
-from CustomInstrument import CustomInstrument
-from CustomSignal import CustomSignal
-from CustomPhotosphere import CustomPhotosphere
-from CustomPrior import CustomPrior
+from .CustomInstrument import CustomInstrument
```

Comment:
It doesn't work with the `.` in front of the Custom class imports. I get two types of errors. Either:
```
Traceback (most recent call last):
  File "main.py", line 11, in <module>
    from .CustomInstrument import CustomInstrument
ValueError: Attempted relative import in non-package
```
or 
```
Traceback (most recent call last):
  File "/Users/sebastien/Research/XPSI_RUNS/PY3_TESTS/examples_fast/Modules/main.py", line 11, in <module>
    from .CustomInstrument import CustomInstrument
ImportError: attempted relative import with no known parent package
```
Depending on whether I try to run the main from the original xpsi directory, or copy the examples directory elsewhere and try to run the main.

It works fine when I remove the `.`

Reply:
I am a strong supporter of absolute rather than relative imports in packages in general, though I'm not sure how that would work here.